src/main/java/org/radarbase/hdfs/config/NonEmptyValidator.kt

```diff
@@ -14,18 +14,17 @@
  * limitations under the License.
  */
 
-package org.radarcns.hdfs.util.commandline;
+package org.radarbase.hdfs.config
 
-import com.beust.jcommander.IParameterValidator;
-import com.beust.jcommander.ParameterException;
+import com.beust.jcommander.IParameterValidator
+import com.beust.jcommander.ParameterException
 
-public class NonEmptyValidator implements IParameterValidator{
-    @Override
-    public void validate(String name, String value) throws ParameterException {
-        if (value == null || value.isEmpty()) {
-            throw new ParameterException("Parameter " + name + " should be supplied. "
-                    + "It cannot be empty or null. (found " + value +")."
-                    + "Please run with --help or -h for more information.");
+class NonEmptyValidator : IParameterValidator {
+    override fun validate(name: String, value: String?) {
+        if (value?.isEmpty() == true) {
```

Comment:
redundant `== true`?
Also why not use `value.isNullOrEmpty()`

Reply:
You're right I should've used `isNullOrEmpty()` instead. `== true` indeed checks is null OR is empty, so the check is not redundant.